pyvo/io/uws/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-__all__ = ['parse_job', 'parse_job_list', 'JobFile', 'JobList']
+__all__ = ['parse_job', 'parse_job_list', 'JobFile', 'Jobs']
```

Comment:
Technically this is a public API change.

However, for some reasons, it's never ended up in [public API docs](https://pyvo.readthedocs.io/en/stable/io/uws.html?highlight=joblist), it was never added into the `__all__` of its actual namespace. 

So, overall I think it's OK to make this change without deprecation, BUT do list `Jobs` in the API here only if you also add it to `__all__` in `tree.py`, otherwise the situation about the public API intent stays somewhat confusing.

Reply:
Good point. I don't think about it because I thought that `JobList`/`Jobs` is not meant to be part of the API as `parse_job_list` is and this function returns just the list of jobs and not the container class. To be technically correct it's probably simpler keep the `JobList` name but I agree that it needs to be added to the `__all__` in `tree.py`.

Reply:
It's OK not being in the public API, but then remove from the `__all__` in this file. I'm just basically advocating for consistency (and pretty sure that some tools should be out there that would warn about this undefined `Jobs/Joblists` here, in fact surprised that it didn't generate a sphinx warning).

Reply:
The naming of things is one thing, normally you would need a rename deprecation. But this was a bug all along, so I think we don't need it here.

However, whether deprecating or not, it still doesn't solve the issue with the namespace, `Jobs` is listed in `__all__`, yet it's not importable.


```
In [1]: from pyvo.io.uws import Jobs
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-c5ca393952e0> in <module>
----> 1 from pyvo.io.uws import Jobs

ImportError: cannot import name 'Jobs' from 'pyvo.io.uws' (/Users/bsipocz/munka/devel/pyvo/pyvo/io/uws/__init__.py)
```